src/key/key.js

```diff
@@ -257,8 +257,13 @@ class Key {
    * @async
    */
   async getSigningKey(keyID = null, date = new Date(), userID = {}, config = defaultConfig) {
-    await this.verifyPrimaryKey(date, userID, config);
     const primaryKey = this.keyPacket;
+    try {
+      await this.verifyPrimaryKey(date, userID, config);
```

Comment:
I wouldn't wrap this function's errors, since all of them already mention the primary key, so it seems a bit redundant